src/xdocs/config_misc.xml

```diff
@@ -302,21 +302,21 @@ String unitAbbrev = "\u03bc\u03bc\u03bc";
         <p>Example #2: Comment is placed at the end of the block and has previous statement.</p>
         <source>
 1   public void foo1() {
-2       foo2();
-3       // it is OK
+2     foo2();
+3     // it is OK
 4   }
 5
 6   public void foo2() {
-7       foo3();
-8          // violation (comment should have the same indentation level as line 7)
+7     foo3();
+8       // violation (comment should have the same indentation level as line 7)
 9   }
         </source>
-        <p>Example #3: Comment is used as a single line border to separate groups of methods.</p>
+<p>Example #3: Comment is used as a single line border to separate groups of methods.</p>
```

Comment:
Please restore indentation on this , there is similar line below

Reply:
Done